exotica_examples/resources/configs/example_time_indexed_sampling.xml

```diff
@@ -8,17 +8,19 @@
   </TimeIndexedRRTConnectSolver>
 
   <TimeIndexedSamplingProblem Name="MyProblem">
-    <T>3</T>
+    <!-- <Debug>1</Debug> -->
```

Comment:
Remove comment. Although we need an example of this. I always ave to look into code to see where the <Debug> tag goes and ig it needs caps.